src/wlcsim/initcond.f03

```diff
@@ -445,23 +445,35 @@ subroutine initcond(R,U,NT,NP,FRMFILE,rand_stat, wlc_p)
         IB=otherEnd+1
     enddo
 elseif (WLC_P__INITCONDTYPE == 'nucleosome') then
-    R(1,1) = 0.0_dp
-    R(2,1) = 0.0_dp
-    R(3,1) = 0.0_dp
+    if (WLC_P__NEIGHBOR_BINS) then 
+        R(1,1) = WLC_P__LBOX_X/2
+        R(2,1) = WLC_P__LBOX_Y/2
+        R(3,1) = WLC_P__LBOX_Z/2
+    else if (WLC_P__CONFINETYPE=='sphere') then 
+        R(1,1) = 3*WLC_P__LBOX_X/4
+        R(2,1) = 3*WLC_P__LBOX_Y/4
+        R(3,1) = -WLC_P__LBOX_Z/4
+    else
+        R(1,1) = 0.0_dp
+        R(2,1) = 0.0_dp
+        R(3,1) = 0.0_dp
+    endif
     U(1,1) = 0.0_dp
     U(2,1) = 0.0_dp
     U(3,1) = 1.0_dp
     wlc_V(1,1) = 1.0_dp
     wlc_V(2,1) = 0.0_dp
     wlc_V(3,1) = 0.0_dp
 
-    do IB=2,WLC_P__NT
+    ! NP EDIT : initialization is at odds with how energy is calculated 
+    ! (i.e i-1 rotation to i vs i rotation to i+1, intuitvely the same physics but not the same results)
+    do IB=1,WLC_P__NT-1
         ! Rotation (and translation) due to nucleosome
-        call nucleosomeProp(U(:,IB-1), wlc_V(:,IB-1), R(:,IB-1), &
+        call nucleosomeProp(U(:,IB), wlc_V(:,IB), R(:,IB), &
```

Comment:
Was there a -1 error?

Reply:
im not entirely sure. when i did the -1 initialization, the chain seemed to quickly sample out/around this configuration, so it made me think that wasn't the "true" 0 energy structure. when i do +1 initialization, the initial configuration seems to be more happy sampling this state. 
honestly im not sure what happened and maybe i was just seeing things, but either way, the initialization should result in the same behavior.